src/css/base.scss

```diff
@@ -3,26 +3,59 @@ $primary-background: radial-gradient(circle, rgba(63,94,251,1) 0%, rgba(252,70,1
 ;
```

Comment:
May be best (for modularity and per project spec) to break this file up into components of the UI and import them into this base file.